lib/appoptics_apm/xtrace.rb

```diff
@@ -47,7 +60,7 @@ def unset_sampled(xtrace)
       # Extract and return the task_id portion of an X-Trace ID
       #
       def task_id(xtrace)
-        return nil unless AppOpticsAPM::XTrace.valid?(xtrace)
+        return nil unless ok?(xtrace)
 
         xtrace[2..41]
```

Comment:
i liked this syntax in python (it uses `:` vs `..`) but it's much nice than javascripts `.slice()`

Reply:
in Ruby there is `..` and `...`, the first includes the last index and `...` doesn't  (it is so twisted) 
`[0,1,2,3,4,5,6][1..3]`
`=> [1, 2, 3]`
`[0,1,2,3,4,5,6][1...3]`
`=> [1, 2]`